examples/pytorch/schnet/Alchemy_dataset.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding:utf-8 -*-
+"""Example dataloader of Tencent Alchemy Dataset 
+https://alchemy.tencent.com/
+"""
+import os.path as osp
+import networkx as nx
+from rdkit import Chem
+from rdkit.Chem import ChemicalFeatures
+from rdkit import RDConfig
+import dgl
+from dgl.data.utils import get_download_dir
+from dgl.data.utils import download
+from dgl.data.utils import extract_archive
+import torch
+from torch.utils.data import Dataset
+from torch.utils.data import DataLoader
+from collections import namedtuple
+import pathlib
+import pandas as pd
+import numpy as np
+
+_urls = {'Alchemy': 'https://alchemy.tencent.com/data/'}
+
+
+class AlchemyBatcher:
+    def __init__(self, graph=None, label=None):
+        self.graph = graph
+        self.label = label
+
+
+def batcher(device):
```

Comment:
It seems `device` is not used in batcher function.
```suggestion
def batcher():
```
Plus, we recommend users to move data to GPU in main functions, collate function should only include CPU operations.

Reply:
Thank u for pointing this out. Forget to delete it before. I have fixed the device-related problems in the next commit.